executor/common_linux.h

```diff
@@ -206,6 +206,37 @@ static int netlink_send(struct nlmsg* nlmsg, int sock)
 	return netlink_send_ext(nlmsg, sock, 0, NULL);
 }
 
+// TODO: switch other methods to netlink_query_family_id
```

Comment:
Will this be resolved in this PR?

Reply:
It was not planned to resolve it in this PR. This PR focuses on changes that are necessary to fuzz 802.11, while this TODO is about general refactoring that will additionally require testing of the affected subsystems.

Reply:
Hmm, I see 2 almost identical functions: netlink_wireguard_id_get, netlink_devlink_id_get.

If all these functions are essentially identical, I don't see any harm in adding a commit at the end of this PR that replaces uses of those with your more generic function that just get passed the right parameter.

Some things to keep in mind:
- There might be some careful #if needed to make sure the generic function is compiled for the configs where we need it.
- Maybe make the error messages print the family_name, as that's what seem to be the main difference between them.

Also, what would make the whole thing more reviewable is simply having another preparatory commit before this commit that simply does the refactor and adds this generic function. That way, it should be obvious from the commit that the functions being replaced are identical with the generic one.

If it turns out it's not that trivial and I missed something, then feel free to leave alone.